docs/time/index.rst

```diff
@@ -489,8 +489,8 @@ To apply arithmetic methods to |Time| instances::
 
   >>> t.max()
   <Time object: scale='utc' format='mjd' value=50002.5>
-  >>> t.ptp(axis=0)  # doctest: +FLOAT_CMP
```

Comment:
I used `Time.min` as a replacement because, as we know, reprs of numpy scalars are different in numpy 1 VS 2, making `min` the only method other than max which result is identical for both versions.

Reply:
Might have used `.argsort` or so myself, but really, that's not worth another CI run... So, fine with this, but will wait with merging until tomorrow so @taldcroft can have a look, if he wishes.